content-scripts/inject/run-userscript.js

```diff
@@ -1,7 +1,10 @@
 import Addon from "../../addon-api/content-script/Addon.js";
+const ranAddons = [];
 
-export default async function runAddonUserscripts({ addonId, scripts, traps }) {
-  const addonObj = new Addon({ id: addonId, traps });
+export default async function runAddonUserscripts({ addonId, scripts, traps }, { late = false } = {}) {
```

Comment:
What does "late" mean?

Reply:
It wasn't enabled on document start, thereby, late.